frontend/src/pages/person/_itemId/index.vue

```diff
@@ -125,98 +126,109 @@ import { getBlurhash } from '@/utils/images';
 import { useRemote, useDateFns } from '@/composables';
 
 const route = useRoute();
-// @ts-expect-error - itemId DOES exist
-const itemId: string = route.params.itemId;
 const remote = useRemote();
 
-const item = (
-  await remote.sdk.newUserApi(getUserLibraryApi).getItem({
-    userId: remote.auth.currentUserId || '',
-    itemId
-  })
-).data;
-
-const movies =
-  (
-    await remote.sdk.newUserApi(getItemsApi).getItems({
-      personIds: [itemId],
-      sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
-      sortOrder: [SortOrder.Descending],
-      recursive: true,
-      includeItemTypes: [BaseItemKind.Movie],
-      fields: Object.values(ItemFields),
-      userId: remote.auth.currentUserId
-    })
-  ).data.Items || [];
-
-const series =
-  (
-    await remote.sdk.newUserApi(getItemsApi).getItems({
-      personIds: [itemId],
-      sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
-      sortOrder: [SortOrder.Descending],
-      recursive: true,
-      includeItemTypes: [BaseItemKind.Series],
-      fields: Object.values(ItemFields),
-      userId: remote.auth.currentUserId
-    })
-  ).data.Items || [];
-
-const books =
-  (
-    await remote.sdk.newUserApi(getItemsApi).getItems({
-      personIds: [itemId],
-      sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
-      sortOrder: [SortOrder.Descending],
-      recursive: true,
-      includeItemTypes: [BaseItemKind.Book],
-      fields: Object.values(ItemFields),
-      userId: remote.auth.currentUserId
-    })
-  ).data.Items || [];
-
-const photos =
-  (
-    await remote.sdk.newUserApi(getItemsApi).getItems({
-      personIds: [itemId],
-      sortBy: ['PremiereDate', 'ProductionYear', 'SortName'],
-      sortOrder: [SortOrder.Descending],
-      recursive: true,
-      includeItemTypes: [BaseItemKind.Photo],
-      fields: Object.values(ItemFields),
-      userId: remote.auth.currentUserId
-    })
-  ).data.Items || [];
-
+const item = ref<BaseItemDto>({});
+const movies = ref<BaseItemDto[]>([]);
+const series = ref<BaseItemDto[]>([]);
+const books = ref<BaseItemDto[]>([]);
+const photos = ref<BaseItemDto[]>([]);
 const activeTab = ref(4);
 
-// Used to pick the first tab with content to display
-if (movies?.length) {
-  activeTab.value = 0;
-} else if (series?.length) {
-  activeTab.value = 1;
-} else if (books?.length) {
-  activeTab.value = 2;
-} else if (photos?.length) {
-  activeTab.value = 3;
-}
-
 const birthDate = computed(() =>
-  item.PremiereDate
-    ? useDateFns(format, new Date(item.PremiereDate), 'PPP').value
+  item.value.PremiereDate
+    ? useDateFns(format, new Date(item.value.PremiereDate), 'PPP').value
     : undefined
 );
 
 const deathDate = computed(() =>
-  item.EndDate
-    ? useDateFns(format, new Date(item.EndDate), 'PPP').value
+  item.value.EndDate
+    ? useDateFns(format, new Date(item.value.EndDate), 'PPP').value
     : undefined
 );
```

Comment:
It seems those values are now undefined

Reply:
It broke from the DateFns change, I've reverted that and applied the fix specifically in the components consuming it, which has also fixed these not displaying. I'm surprised this didn't cause a type error thought :S